packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -274,6 +274,11 @@ class ScrollDragController implements Drag {
   static const Duration momentumRetainStationaryDurationThreshold =
       Duration(milliseconds: 20);
 
+  /// The factor multiplied with the carriedVelocity to check if it should be
```

Comment:
I didn't understand this sentence :D 

Feel free to use longer/multiple sentences to describe it. Also, an example can help illustrate if needed.

Reply:
Rephrased this and added an example